ee/clickhouse/client.py

```diff
@@ -12,7 +12,7 @@
 )
 
 if PRIMARY_DB != CLICKHOUSE:
-    ch_client = None
+    ch_client = Client()
```

Comment:
@fuziontech made this an empty client so that type checking doesn't fail lmk if this is a problem

Reply:
This is perfect